nix/commands/make.sh

```diff
@@ -0,0 +1 @@
+build.sh
```

Comment:
Is this a symlink or a file? Hard to tell on github UI

Reply:
both bat and sh are symlinks